compiler/msgs.nim

```diff
@@ -715,7 +715,10 @@ type
 proc handleError(msg: TMsgKind, eh: TErrorHandling, s: string) =
   template quit =
     if defined(debug) or gVerbosity >= 3 or msg == errInternal:
-      writeStackTrace()
```

Comment:
So use strutils.countLines(st) > 1: st else: "..." instead. Be more creative please. ;-)